packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -173,10 +173,12 @@ export class SmoothControls extends EventDispatcher {
           'touchstart', this.onTouchStart, {passive: true});
       element.addEventListener('touchmove', this.onTouchMove, {passive: false});
 
+      element.addEventListener('touchend', this.onTouchEnd);
```

Comment:
We can attach the `touchend` listener to `element` instead of `self`. We can afford to do this because as observed [here](https://github.com/google/model-viewer/issues/2682#issuecomment-896964938), `touchmove` and `touchend` events fire on the target at `touchstart` time, even if that touch leaves the target's bounds.
Since all our event listeners are now attached to `element`, we can now react to `targetTouches` instead of `touches`. `targetTouches` are more suitable because we are not interested in touches which do not start on `element`.
This also helps avoid littering the global object with event listeners.